programmers/string-handling-basic.js

```diff
@@ -0,0 +1,20 @@
+function solution(s){
+    const linerMeasure = s.length;
+    if(linerMeasure === 4 | linerMeasure === 6){
+        for (let i = 0 ; i<linerMeasure; i++){
+            let store = 0;
+            for (let j = 0 ;  j<10 ; j++){
+                if (s[i] == j){
+                    store = 1;
+                }
+            }
+            if (store === 0){
+                return console.log("false");
+            }
+        }
+        return console.log("true");
+    }
```

Comment:
정규표현식을 이용해서 간단하게 풀수도 있습니다.

```suggestion
function solution(s) {
    if((s.length == 4 || s.length == 6) && /^[0-9]+$/.test(s)) {
        return true;
    }
    return false;
}
```